Assets/Scripts/GameService.cs

```diff
@@ -75,4 +76,9 @@ private void Update()
     public UIView GetUIService() => uiService; 
     #endregion
 
+    public void ShakeCamera()
+    {
+        cameraShake.Shake();
+    }
```

Comment:
Why is Camera Shake method in GameService, GameService is a service locator, not a camera controller. we create more classes, as many as we want but will always maintain SRP